.circleci/config.yml

```diff
@@ -0,0 +1,24 @@
+version: 2
+jobs:
+  build_all:
+    working_directory: ~/airbrake-python-integrations
+    docker:
+      - image: randomknowledge/docker-pyenv-tox
+
+    steps:
+      - checkout
+
+      - run:
+          name: install
+          command: |
+            pip install -U pip virtualenv
+            virtualenv venv
+            source venv/bin/activate
+            pip install tox
+            tox -v --recreate
+
+workflows:
```

Comment:
No need to use workflows if we have only 1 job. Just name your job `build` on line 3 and you can safely delete all of this section.

Reply:
I think the intention is to setup us up to use multiple jobs _if_ we decide to tests multiple against multiple pythons in parallel.

Let's merge this and revisit as this will need to change in airbrake-python as well.